wallet/udb/addressmanager.go

```diff
@@ -1294,6 +1294,51 @@ func (m *Manager) ImportPrivateKey(ns walletdb.ReadWriteBucket, wif *dcrutil.WIF
 	return managedAddr, nil
 }
 
+// pubkey has to be serialized pubkey (serialized compressed SEC: y sign + 32B BigEndian x)
+func (m *Manager) ImportPublicKey(ns walletdb.ReadWriteBucket, pubkey []byte) (ManagedPubKeyAddress, error) {
+	defer m.mtx.Unlock()
+	m.mtx.Lock()
+
+	// The manager must be unlocked to encrypt the imported private key.
+	if !m.watchingOnly && m.locked {
+		return nil, errors.E(errors.Locked)
+	}
+
+	// Prevent duplicates.
+	serializedPubKey := pubkey
+	pubKeyHash := dcrutil.Hash160(serializedPubKey)
+	if existsAddress(ns, pubKeyHash) {
+		return nil, errors.E(errors.Exist, "address for private key already exists")
```

Comment:
Looks like this error message wouldn't necessarily be accurate (given implementation in this PR), same holds for similar error message in [ImportPrivateKey method](https://github.com/decred/dcrwallet/blob/7cdfb44a493995982b6263eceba698d661fbf821/wallet/udb/addressmanager.go#L1268-L1268)
```
address for private key already exists
```

If I understand correctly "how imported accounts work", only one imported account is supported,
and if **I import pubkey account** -> **and then want to replace it with privkey account** **(corresponding to the same public key)** dcrwallet [won't allow me to do that](https://github.com/decred/dcrwallet/blob/7cdfb44a493995982b6263eceba698d661fbf821/wallet/udb/addressmanager.go#L1268-L1268) (and this error [will even be silenced](https://github.com/decred/dcrwallet/blob/7cdfb44a493995982b6263eceba698d661fbf821/internal/rpc/jsonrpc/methods.go#L2068-L2070)); I don't know what's the proper/best way to handle it - just pointing out what I've noticed